docs/user-assigned-msi-mode.md

```diff
@@ -4,7 +4,7 @@
 
 This option allows azure KeyVault to use the user assigned managed identity on the k8s cluster VMSS directly.
 
-AKS does not support user assigned managed identities yet, only system assigned managed identities. Until this gap is covered, use a service principal or system assigned identities with AKS.
+In AKS you can use the user assigned Kublet managed identity (doesn't support BYO today) or create your own user assigned managed identity as described below.
```

Comment:
nit: s/Kublet/Kubelet

Reply:
I don't think it's obvious that `user assigned Kubelet managed identity` here means `system assigned managed identity`. Can we use the same terminology across all these pages?

Reply:
@ritazh We have that in the doc already -

```
You can create AKS with managed identities now and then you can skip steps 1 and 2. To be able to get the CLIENT ID, the user can run the following command

az aks show -g <resource group> -n <aks cluster name> --query identityProfile.kubeletidentity.clientId -o tsv
```
Do you think we need any other change?

@TomGeske one small nit: s/Kublet/Kubelet

Reply:
fixed the typo

Reply:
@ritazh: I just added a link to our MI docs where we mention kubelet MI.